mungegithub/submit-queue/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: k8s-submit-queue
+spec:
+  rules:
+  - host: kubernetes.submit-queue.k8s.io
+    http:
+      paths:
+        - backend:
+            serviceName: k8s-kubernetes-sq-status
```

Comment:
I was going with `orgname-project` name syntax. I can change it to just project if that is better.